pkg/controllers/dnspolicy/dns_helper.go

```diff
@@ -124,7 +124,7 @@ func (dh *dnsHelper) buildDNSRecordForListener(gateway *gatewayv1beta1.Gateway,
 }
 
 // getDNSRecordForListener returns a v1alpha1.DNSRecord, if one exists, for the given listener in the given v1alpha1.ManagedZone.
-// It needs a reference string to enforce DNS record serving a single traffic.Interface owner
+// It needs a reference string to enforce DNS record serving a single utils.Interface owner
```

Comment:
You could probably just remove this line, I've read it 5 times and i don't know what it means, maybe meant something once :shrug: .

Reply:
removed